sklearn/svm/_classes.py

```diff
@@ -242,6 +262,10 @@ def fit(self, X, y, sample_weight=None):
         self : object
             An instance of the estimator.
         """
+        self._validate_params()
+
+        check_random_state(self.random_state)
+
```

Comment:
As mentioned in a prior comment, this doesn't actually validate the random state, we do that by specifying the special `"random_state"` constraint. This builds a random state, and so is often used in e.g. `fit`. Not needed here.

```suggestion
```